### PR TITLE
Adding Eviction to glossary

### LINKED ISSUE #29223: Add Eviction to glossary
**This is a Feature Request**

**What would you like to be added**
Add an entry to the Kubernetes glossary, “Eviction”. Explain how Kubernetes uses this term. There are two flavors of eviction: node-initiated eviction, and API-initiated eviction. The detailed glossary entry should mention both and hyperlink to more detailed articles on these.

**Why is this needed**
“eviction” is Kubernetes jargon; readers may well come across the term without recognising it.

We have entries for the specific flavors but not an overall entry for the general term.

**Comments**
Also see issue #14507
/kind feature
/language en
/priority backlog

### PR DESCRIPTION
**Pull request by niteshseram**
Closes #29223

### CODE REVIEW

**Comment by jihoon-seo** [comment]
Pinging @sftim

File: content/en/docs/reference/glossary/eviction.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+---
+title: Eviction
+id: eviction
+date: 2021-05-08
+full_link: /docs/concepts/scheduling-eviction/
+short_description: >
+    Process of terminating one or more Pods on Nodes
```
Comment (bradtopol) [inline content/en/docs/reference/glossary/eviction.md:7]: great job! Definition is correct, content renders perfectly and links are valid.

**Comment by bradtopol** [comment]
Great job!
/lgtm
/approve

**Pull request merged**